bucket4j-spring-boot-starter-context/src/main/java/com/giffing/bucket4j/spring/boot/starter/context/ExpressionParams.java

```diff
@@ -12,17 +12,19 @@
  *
  * @param <R> the type of the root object which us used for the SpEl expression.
  */
+@Getter
 @RequiredArgsConstructor
 public class ExpressionParams<R> {
 
-    @Getter
+    public static final String IP = "ip";
```

Comment:
I don't like the additional IP related stuff. I'ts a good idea to make to simplify the access to the IP address but i would prefer a more general solution for this. I haven't added it on purpose yet because it may depends on the users use case. I would prefer to remove it and focus on the metric filters.

Reply:
The IP related stuff was removed here.